content/en/aws/elastic-compute-cloud/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Elastic Compute Cloud (EC2)"
+linkTitle: "Elastic Compute Cloud (EC2)"
+categories: ["LocalStack Pro"]
+description: Elastic Compute Cloud (EC2)
+---
+
+LocalStack has basic support for two backend managers for running instances locally: `virtualbox` (default) and `docker`.
+The manager can be configured by using the `EC2_VM_MANAGER` environment variable.
+
+### VirtualBox
+
+The VirtualBox backend requires the [Oracle VM VirtualBox](https://www.virtualbox.org/) installed on your system.
+
+<!-- Where is the stuff about EC2 local daemon documented? Link it here? -->
```

Comment:
Good point, I don't think we have that documented yet! We could add a short note like:
```
To leverage the full functionality of the EC2 VirtualBox emulation, it is necessary to run a daemon process on the host which takes care of creating and managing the VirtualBox VM instances. You can use the LocalStack command-line interface to start up the daemon:
``
$ export LOCALSTACK_API_KEY=...
$ localstack daemons start
...
``

Depending on your host operating system, you may be asked to enter your `sudo` password. This is required to create virtual network interfaces on the local machine with IP addresses that allow to easily SSH into the created VirtualBox instances.
```